tests/sentry/issues/test_ingest.py

```diff
@@ -183,8 +183,8 @@ def test(self) -> None:
 @region_silo_test
 class SaveIssueOccurrenceToEventstreamTest(OccurrenceTestMixin, TestCase):  # type: ignore
     def test(self) -> None:
-        # TODO: We should make this a platform event once we have one
-        event = self.store_event(data={}, project_id=self.project.id)
+        event_data = load_data("generic-event")
+        event = self.store_event(data=event_data, project_id=self.project.id)
```

Comment:
should this be creating a group? I thought it would but since it's going through eventstream it looks like it isn't, perhaps only through kafka

Reply:
aha yes. we create the group from the issue occurrence, not the event. so this test won't pass without updating

Reply:
Yeah, you shouldn't get a group until you run through the issue occurrence flow